.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 22.3.0
```

Comment:
Is there are reason to change this in this pull request? I recall the black project did make a change to their output recently, so perhaps you could provide a separate pull request for this instead?